cellarium/cas/exceptions.py

```diff
@@ -41,3 +41,7 @@ class DataValidationError(CASBaseError):
 
 class QuotaExceededError(CASBaseError):
     pass
+
+
+class ClientTooOldError(CASBaseError):
```

Comment:
I'm torn: should this be more generic?  Like `InvalidClient`?

Reply:
Why more generic?  So it can be used in the future for other things that might make the client incompatible with the server?

Reply:
Yeah, I was thinking in case there are other types of version constraints that could be violated.  Eh, I think it's likely that even a buggy client version would be just fixed with a bump so it seems reasonable to just leave as is